### ISSUE TITLE
Can you please tell what the right answer is, when the user gets it wrong after 10 tries?

### ISSUE THREAD

**Issue by Puneet-G** [wordle-hindi/wordle-hindi.github.io#1]
Can you please tell what the right answer is, when the user gets it wrong after 10 tries?

I use your site, https://wordle-hindi.github.io/, regularly, but when you get the answer wrong, it does not tell you what the word was.

**Comment by Puneet-G** [comment]
Also, some days I think there is no right answer, its happened twice now, or maybe I just don't know enough words.
![image](https://github.com/user-attachments/assets/31aec0ac-535f-4e34-a460-fc88193a07fe)

**Comment by VedantSahu01** [comment]
> Also, some days I think there is no right answer, its happened twice now, or maybe I just don't know enough words.
> ![image](https://github.com/user-attachments/assets/31aec0ac-535f-4e34-a460-fc88193a07fe)
> 

Yes, i am aware of this issue, but not getting time to fix this😅. Will try to fix this and add some more features by the end of this year

**Comment by Puneet-G** [comment]
Thanks!
Looking forward to the changes!
